src/xenia/gpu/gl4/draw_batcher.cc

```diff
@@ -289,6 +290,8 @@ bool DrawBatcher::Flush(FlushMode mode) {
         // assert_true(
         // (register_file_->values[XE_GPU_REG_PA_SU_SC_MODE_CNTL].u32
         // & 0x3) == 0);
+        cull_enabled = glIsEnabled(GL_CULL_FACE);
```

Comment:
:scream: queries are really, really bad -- they seem like they'd be fast, but they never are. But, this code will all die eventually, so it's fine!